src/Utilities/TextArea/index.tsx

```diff
@@ -0,0 +1,22 @@
+interface TextareaProps {
+  name: string;
+  id: string;
+  title: string;
+  required?: boolean;
+}
+
+export default function Textarea({ name, id, title, required }: TextareaProps) {
```

Comment:
```suggestion
export default function TextAreaInput({ name, id, title, required }: TextAreaProps) {
```

Can't use TextArea obv because that conflicts with the textarea tag, but we should avoid naming it the-same-except-for-casing too - it's confusing.

Reply:
Done